tests/chart_tests/test_astronomer_houston_task_metrics.py

```diff
@@ -9,6 +9,35 @@
     supported_k8s_versions,
 )
 class TestAstronomerHoustonTaskMetricsCronjobs:
+    def test_astronomer_hourly_audit_task_metrics_cron_defaults(self, kube_version):
+        docs = render_chart(
+            kube_version=kube_version,
+            values={"global": {"taskUsageMetricsEnabled": False}},
+            show_only=[
+                "charts/astronomer/templates/houston/cronjobs/houston-populate-hourly-task-audit-metrics.yaml",
+            ],
+        )
+        assert len(docs) == 0
```

Comment:
We should make more assertions here, such as things that should not exist by default.